requirements.docs.txt

```diff
@@ -4,4 +4,5 @@ numpydoc
 sphinx-rtd-theme
 notedown
 jsx-lexer
-eralchemy==1.2.10
+# need a fix for SQLA1.4
```

Comment:
this codebase hasn't been update in many years. This PR fixes the issues introduced in SQLA1.4.